packages/expo-router/src/modal/web/ModalStack.web.tsx

```diff
@@ -74,8 +75,21 @@ const ModalStackView = ({ state, navigation, descriptors, describe }: ModalStack
       {isWeb &&
         state.routes.map((route, i) => {
           const isModalType = isModalPresentation(descriptors[route.key].options);
-          const isActive = i === state.index && isModalType;
-          if (!isActive) return null;
+          const isActive = i === 0 || !isModalType;
+          if (isActive) return null;
```

Comment:
Why return `null` fro `i===0`?

Reply:
Is it in case modal is first and is not rendered as modal?

Reply:
This was a test leftover

Reply:
Should be fixed, can you test one more time? Feel free to push directly as well!